src/test/java/com/github/skjolberg/packing/Visualizer.java

```diff
@@ -24,7 +24,7 @@ public static String visualize(Container container, int size, double horizontalS
 			IContextBuilder builder = render.newBuilder();
 			
 			int w = (int)(size * horizontalScaling);
-			int d = (int)((size * container.getDepth()) / container.getWidth());
```

Comment:
I'd like to keep this so that the multiplication is performed before the division.

Reply:
well, the parenthesis don't change the semantics here, they have same priority so evaluated left to right

Reply:
Ok, I though that divider was before multiplication, but they're equal and from the left. So the updated code is correct.